src/runtime/analytics-service.js

```diff
@@ -53,6 +54,9 @@ export class AnalyticsService {
     /** @private @const {string} */
     this.publicationId_ = deps.pageConfig().getPublicationId();
 
+    /** @private {?string} */
+    this.transactionId_ = uuidFast();
```

Comment:
According to this, it's never `?`

Reply:
Fixed. Originally, I had set the initial value to `null`;